be/src/storage/rowset/vectorized/segment_iterator.cpp

```diff
@@ -235,7 +231,7 @@ class SegmentIterator final : public ChunkIterator {
     vectorized::SegmentReadOptions _opts;
     RawColumnIterators _column_iterators;
     ColumnDecoders _column_decoders;
-    std::vector<BitmapIndexIterator*> _bitmap_index_iterators;
+    std::unordered_map<ColumnId, std::unique_ptr<BitmapIndexIterator>> _bitmap_index_iterators;
```

Comment:
It's good to replace `std::vector` with `std::unordered_map`, especially when the total number of columns is large, `std::unordered_map` may use less memory.

Reply:
Yes, I plan to optimize `Status new_iterator(ColumnIterator** iterator)` in next PR, thanks!